acceptance_tests/README.md

```diff
@@ -0,0 +1,10 @@
+BRP Acceptance Tests
+
+Install:
+
+Bundle install the Gem file 

```

Comment:
perhaps worth updating this to include `rvm install ruby-2.1.0-preview2` && `rvm use 2.1.0` in case the user hasn't got Ruby installed, we'll also need to install Ruby in Travis (CI) in order to run these tests.

Sire note: when printing a command, it's really helpful if you quote the command in backticks (`) so this;

Bundle install the Gem file becomes `Bundle install the Gem file`